hackclub.com.yaml

```diff
@@ -2855,3 +2855,8 @@ syria:
   - ttl: 600
     type: CNAME
     value: hackclub-syria.github.io.
+
+astra-collective:
+  - ttl: 600
+    type: CNAME
+    value: https://astra-collective.github.io/ASTRA-Website/.
```

Comment:
DNS records can't have protocols (`https://`) or paths (`/ASTRA-Website/`) in them